### PR TITLE
feat: implemented LQL Queries/Policies endpoints

### PR DESCRIPTION
**Pull request by alannix-lw**

### CODE REVIEW

File: laceworksdk/api/policies.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+               title,
+               description,
+               remediation,
+               severity,
+               alert_enabled,
+               alert_profile,
+               evaluator_id,
+               limit=None,
+               eval_frequency=None,
+               policy_ui={},
```
Comment (hazedav) [inline laceworksdk/api/policies.py:39]: nit.  Setting default method variables to immutable types in python is an anti-pattern.  I would recommend setting this to `None` and then on the first line:

`policy_ui = {} if policy_ui is None else policy_ui`

File: laceworksdk/api/policies.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+            "description": description,
+            "remediation": remediation,
+            "severity": severity,
+            "alertEnabled": int(bool(alert_enabled)),
+            "alertProfile": alert_profile,
+            "policyUi": policy_ui,
+            "evaluatorId": evaluator_id
+        }
+
+        if limit:
```
Comment (hazedav) [inline laceworksdk/api/policies.py:84]: What if we want to set `limit` to zero here?  In that case `limit` would be falsy and this code skipped.  I would recommend:

`if isinstance(limit, int) and limit>=0:`

File: laceworksdk/api/policies.py [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+            data["description"] = description
+        if remediation:
+            data["remediation"] = remediation
+        if severity:
+            data["severity"] = severity
+        if alert_enabled is not None:
+            data["alertEnabled"] = bool(alert_enabled)
+        if alert_profile:
+            data["alertProfile"] = alert_profile
+        if limit:
```
Comment (hazedav) [inline laceworksdk/api/policies.py:197]: Similar comment regarding limit check here...

**Review by hazedav — APPROVED** [review]
Approved w/ comments

File: laceworksdk/api/policies.py (hunk 1 above)
Comment (alannix-lw) [reply laceworksdk/api/policies.py]: Thanks @hazedav! Always dropping knowledge - I appreciate it!

File: laceworksdk/api/policies.py (hunk 2 above)
Comment (alannix-lw) [reply laceworksdk/api/policies.py]: 👍

File: laceworksdk/api/policies.py (hunk 3 above)
Comment (alannix-lw) [reply laceworksdk/api/policies.py]: 👍

**Comment by alannix-lw** [comment]
Current failing tests are a result of https://lacework.atlassian.net/browse/RAIN-19427 - unrelated to changes in this PR.  Once that issue is corrected, tests should run successfully.

**Pull request merged**